### ISSUE TITLE
Metric Cluster.ActiveRpcWriteCount displayed -1

### ISSUE THREAD

**Issue by maobaolong** [Alluxio/alluxio#13826]
**Alluxio Version:**

Master

**Describe the bug**

I got the negative value of metric Cluster.ActiveRpcWriteCount, guess there are some exception or unexpected break.

**To Reproduce**

Not sure.

**Expected behavior**

Display the Cluster.ActiveRpcWriteCount  state correctly.

**Urgency**
Normal

**Additional context**
No

**Comment by LuQQiu** [comment]
https://github.com/Alluxio/alluxio/pull/13834 for some RPCCounter metrics syntax fix, but is not related to solving this issue.
@ZhuTopher this error could come from a miscalculation of the RPC values?
e.g. 
```
    // Decrement RPC counter only if the request wasn't completed/canceled already
    if (!context.isDoneUnsafe()) {
      RPC_WRITE_COUNT.dec();
    }
```

**Comment by maobaolong** [comment]
@LuQQiu Thanks for your input.

**Comment by ZhuTopher** [comment]
Yeah that's definitely possible that the decrement occurs more than once. I looked into it with @ggezer when I added them to try and avoid them getting decremented more than once per-RPC but wasn't able to test it too thoroughly. Thanks for making this issue!

**Comment by LuQQiu** [comment]
@ZhuTopher Any updates for this?
Basically @maobaolong is asking for a way to understand the worker load status,
If ActiveRead/Write Count, something using metrics increment/decrement is hard to get it right and is easy to break, we can use other formats. Let me know if you have more information about this.
It may be better to directly expose some internal status like thread pool size, etc. 
On the master side, we have metrics like Master.RpcQueueLength (Length of the master rpc queue)

**Comment by ZhuTopher** [comment]
@LuQQiu I haven't touched this ticket, most likely those specific `Counter`s will be deprecated and replaced with something else as those were introduced as a hold-over. Other existing metrics would probably be better indicators of worker stress if we have any.

**Comment by LuQQiu** [comment]
@apc999 @yuzhu any thoughts on which existing metrics can indicate worker stress or what metrics can we add for indicate worker stress?

**Comment by ggezer** [comment]
@LuQQiu high eviction rate would be an indicator for stress on capacity.

**Comment by LuQQiu** [comment]
Thanks @ggezer ! Worker stress includes 
- the capacity part, `Worker.BlocksEvictionRate`
- how many gRPC this worker receives, or how many gRPC request is in queue, or in processing (I haven't been able to find where to add this metrics)
- some latency in serving worker requests
